<launch>
    <param name="result_dir" type="str" value="$(find hexapod_training)/results" />
   <node pkg="hexapod_training" name="result_loader" type="result_manager.py" output="screen"/>
</launch>
